GitHub-Org-Management-Policy.md

```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
 
 ### Owners
 
-TSC members and the Chair of the Community Committee are the only individuals granted
-Owner permissions within the Node.js GitHub Organization.
+As with other resources within the community, Org ownership will be
+granted in order to optimize for the conflicting requirements of:
+
+* Limiting access to reduce risk
+* Enabling individuals to move community work forward without undue delay
+
+When possible, automation and tools should be used to reduce the breadth of
+access that needs to be provided in order to enable individuals to move
+community work forward.  As these tools are developed and put in place we will
+reduce the groups to which org ownership is granted.
+
+At the current time, the following groups are granted access in order for them
+to facilitate community work.
+
+* TSC members
+* Chair of the Community Committee
+* Moderation team members. - It is expected that moderation team members
```

Comment:
Nit: remove the dash/hyphen/whatever.

Reply:
Nit: Get rid of "It is expact that".

Reply:
Generally, I think we've been styling it captialized in the docs: `Moderation Team`